docs/af-fundamentals-tutorial/modules/dlq/pages/implementing.adoc

```diff
@@ -0,0 +1,57 @@
+= Implementing Dead Letter Queues (DLQ)
+:navtitle: Implementing dead letter queue
+
+To enable a DLQ you need to configure the `SequencedDeadLetterQueue` component.
+Like most components related to event processing, there are implementations available for JPA, JDBC, and Mongo.
+To best leverage transactions, it's best to choose the same implementation as the `TokenStore` and likely also for your projection.
```

Comment:
Perhaps this is also the right position to state that not only the implementation types should align, but also the storage location?

Reply:
It can't hurt, I added something about the location.